src/newcharacter.cpp

```diff
@@ -187,6 +187,7 @@ void avatar::randomize( const bool random_scenario, points_left &points, bool pl
     init_age = rng( 16, 55 );
     // if adjusting min and max height from 145 and 200, make sure to see set_description()
     init_height = rng( 145, 200 );
+    blood_type = static_cast<blood_types>( rng( 0, 7 ) );
```

Comment:
To implement `enum_traits` you'll need to add a `last` or `num_blood_types` enumerator in your enum.  Once that exists, I suggest using it here rather than a hardcoded 7.